.github/workflows/haskell-ci.yml

```diff
@@ -0,0 +1,221 @@
+# This GitHub workflow config has been generated by a script via
```

Comment:
Adding CI config generated by haskell-ci.
Let me know if you don't like it - I can remove it.
But I have good experience with it, maintaing packages that need to work with different ghc versions.